packages/contracts/hardhat.config.ts

```diff
@@ -52,6 +52,11 @@ const config: HardhatUserConfig = {
       url: 'http://127.0.0.1:8545',
       accounts: [privateKey],
     },
+    kovan: {
+      chainId: 42,
```

Comment:
I believe we also want

```
    live: true,
    saveDeployments: true,
```

Reply:
This was usually handled by this block of code at the bottom:

```js
if (
  process.env.CONTRACTS_TARGET_NETWORK &&
  process.env.CONTRACTS_DEPLOYER_KEY &&
  process.env.CONTRACTS_RPC_URL
) { ... }
```

These env vars should be deprecated eventually, cc @smartcontracts

Reply:
I'm not using `hardhat-deploy` for these contracts since it assumes that you're deploying all the contracts at once. As a result it's not posting deployments to the deployments folder. I can add this if it's a requirement, I'll just have to figure out how to finagle `hardhat-deploy` to do what I want.

Reply:
You can use a hardhat deploy tag to specify a subset of contracts that you want to deploy. I do think having the artifact is nice, its the way that we enshrine "official" addresses by having them in this gitrepo

Reply:
Hey @smartcontracts can you help out with adding hardhat deploy support?